File: x_pattern.c
#include<stdio.h>
int main(){
	int i,j,l,m,check;
	scanf("%d",&i);
	scanf("%d",&j);
	int a[i][j];
	for (l=0;l<i;l++){
		for (m=0;m<j;m++){
			scanf("%d",&a[l][m]);
		}
	}
	for (l=0;l<i;l++){
		check = j-l-1;
		for (m=0;m<j;m++){
			if(m==check)
				printf("%d",a[l][m]);
			else if(l==m)
				printf("%d",a[l][m]);
			else
				printf(" ");
		}
		printf("\n");
	}
	return 0;
}

File: x_pattern.py
a = int(input())
b = int(input())
mat = []
for i in range(a):
	c = [int(x) for x in input().split()]
	mat.append(c)
for i in range (a):
	check = b-i-1
	for j in range (b):
		if (i==j):
			print(mat[i][j],end="")
		elif(j==check):
			print(mat[i][j],end="")		
		elif (i!=j):
			print(" ",end="")
	print("")
